test/integration/script-loader/test/index.test.js

```diff
@@ -17,31 +17,12 @@ import cheerio from 'cheerio'
 
 let appDir = join(__dirname, '../base')
 let appWithPartytownMissingDir = join(__dirname, '../partytown-missing')
-let appWithStrictModeDir = join(__dirname, '../strictmode')
 let server
 let appPort
 
-async function bootApp(dir) {
-  await nextBuild(dir)
-
-  const app = nextServer({
-    dir,
-    dev: false,
-    quiet: true,
-  })
-
-  server = await startApp(app)
-  appPort = server.address().port
-}
-
-describe('Next.js Script - Primary Strategies', () => {
-  beforeAll(async () => {
-    await bootApp(appDir)
-  })
-  afterAll(() => {
-    stopApp(server)
-  })
-
+const runTests = (isDev = false) => {
+  // TODO: We will refactor the next/script to be strict mode resilient
+  // Don't skip the test case for dev mode (strict mode) once refactoring is finished
```

Comment:
Refactoring the test file in a future PR sgtm. Should we also move it to the `e2e/` folder since all tests there run in both dev and prod?

Reply:
Maybe. But I thought we are still testing the "internal" behavior of `next/script` components here, so "integration test" would probably fit better.